test/setup.js

```diff
@@ -32,6 +32,10 @@ m._load = function hookedLoader(request, parent, isMain) {
         request = './search_bar.ios';
     }
 
+    if (request === 'react-native-linear-gradient') {
```

Comment:
Is there some way to do this automatically for all platform-specific JS files? It seems like we keep having to add those